papers/source/d1664.bs

```diff
@@ -158,22 +164,29 @@ For example `std::ranges::single_view` contains a [exposition *semiregular-box*
 
 ## Applicability ## {#design-applicable}
 
-There are many ranges to which this is applicable, but only a handful in the standard library need or satisfy this. If [[p1391r2]] and [[p1394r2]] are accepted, then the two most important view types -- `std::span<T, Extent>` and `std::basic_string_view<Char, Traits>` -- will have this concept applied to it. `std::ranges::subrange<Iterator, Sentinel, Kind>` already fits this as well. By making it a concept in the standard, we can dependably and reliably assert that these properties continue to hold for the ranges which it is desirable. Some ranges and range customization point objects to which this would be helpfully applicable to in the current standard and proposals space are:
+There are many ranges to which this is applicable, but only a handful in the standard library need or satisfy this. If [[p1391r2]] and [[p1394r2]] are accepted, then the two most important view types -- `std::span<T, Extent>` and `std::basic_string_view<Char, Traits>` -- will model both concepts. `std::ranges::subrange<Iterator, Sentinel, Kind>` already fits this as well. By formalizing concepts in the standard, we can dependably and reliably assert that these properties continue to hold for these ranges. The ranges to which this would be helpfully applicable to in the current standard and proposals space are:
 
-- `view::drop` (currently under consideration, [[p1035r6]] and [[p1739r0]], re-proposed here);
-- `view::take` (currently under consideration, [[p1739r0]], re-proposed here);
-- `std::span` as a reconstructible range (currently under consideration, [[p1394r2]]);
-- `std::basic_string_view` as a reconstructible range (currently under consideration, [[p1391r2]]);
-- `std::ranges::empty_view` as a reconstructible range (proposed here);
-- and, `std::ranges::iota_view` as a reconstructible range (proposed here).
+- `std::ranges::subrange` (already reconstructible)
+- `std::span` (currently under consideration, [[p1394r2]]);
+- `std::basic_string_view` (currently under consideration, [[p1391r2]]);
+- `std::ranges::empty_view` (proposed here);
+- and, `std::ranges::iota_view` (proposed here).
+
+The following range adaptor closure objects will make use of the concepts in determing the type of the returned range:
 
+- `view::drop` (currently under consideration, [[p1035r6]] and [[p1739r0]], re-proposed here);
+- `view::take` (currently under consideration, [[p1739r0]], re-proposed here).
 

```

Comment:
I split this into two blocks, because I perceived it as confusing that things that *are* ranges (*model* the concept) are mixed up with things that *return* ranges (*use* the concept).

Reply:
Sounds good to me!